frontend/src/loaders/submission-overview-loader.ts

```diff
@@ -27,10 +27,16 @@ export default async function loadSubmissionOverview({
 }: {
   params: Params<string>;
 }) {
+

```

Comment:
unnecessary new line

Reply:
It adds to the appearance off the codebase

Reply:
You should format the document with prettier for this! Appearance is something subjective. To keep our code base maintainable and readable by all developers its best we keep ourselves to the conventions and not to our own subjective preferences.

Reply:
No it isn't subjective cause my opinion is absolute.